Modules/posixmodule.c

```diff
@@ -7440,7 +7440,7 @@ win_readlink(PyObject *self, PyObject *args, PyObject *kwargs)
         return NULL;
     }
     print_name = rdb->SymbolicLinkReparseBuffer.PathBuffer +
-                 rdb->SymbolicLinkReparseBuffer.PrintNameOffset;
+                 (rdb->SymbolicLinkReparseBuffer.PrintNameOffset / 2);
```

Comment:
Can `PrintNameOffset` be odd?

Reply:
`PathBuffer` is of type `wchar_t`. Strictly this should use `sizeof(wchar_t)` instead of 2. Alternatively it can use two casts to have the compiler do the arithmetic: `(wchar_t *)((char *)rdb->SymbolicLinkReparseBuffer.PathBuffer + rdb->SymbolicLinkReparseBuffer.PrintNameOffset)`.

Reply:
done